packages/code-gen/src/generator/openAPI/generator.js

```diff
@@ -121,25 +118,77 @@ export function generateOpenApiFile(structure, options) {
 
   // set meta
   openApiSpec.info = {
-    title: `${options.openApiOptions?.title ?? process.env.APP_NAME}`,
-    description: options.openApiOptions?.description ?? "",
-    version: options.openApiOptions?.version ?? "0.0.0",
+    title: `${options.openApiExtensions?.title ?? process.env.APP_NAME}`,
+    description: options.openApiExtensions?.description ?? "",
+    version: options.openApiExtensions?.version ?? "0.0.0",
```

Comment:
Should these values also be nested on `openApiExtensions.info`? If we keep the same structure for the extensions as is expected by OpenApi we can even assign the extensions at the first line of this function `const openApiSpec = Object.assign({}, OPENAPI_SPEC_TEMPLATE, options.openApiExtensions);`. And then this block would only add the default values if none exist

eg. 
```js
openApiSpec.info.version = openApiSpec.info.version ?? "0.0.0";
```